crates/parser/src/token_extension_program/ix_parser/extensions/confidential_transfer_fee.rs

```diff
@@ -159,7 +159,7 @@ mod proto_parser {
         ConfidentialWithdrawWithheldTokensFromMintAccounts, DisableHarvestToMintAccounts,
         EnableHarvestToMintAccounts, InitializeConfidentialTransferFeeConfigAccounts,
     };
-    use crate::helpers::{FromVecPubkeyToVecString, IntoProto};
+    use crate::helpers::{proto_helpers::FromVecPubkeyToVecString, IntoProto};
 
```

Comment:
If this is already in helpers crate then call this module `proto` `helpers::proto::FromVecPubkeyToVecString`